unicodetools/data/ucd/dev/EastAsianWidth.txt

```diff
@@ -30,10 +30,10 @@
 # Character ranges are specified as for other property files in the
 # Unicode Character Database.
 #
-# The comments following the number sign "#" list the General_Category
-# property value or the L& alias of the derived value LC, the Unicode
-# character name or names, and, in lines with ranges of code points,
-# the code point count in square brackets.
+# The comments following the number sign
+# "#" list the General_Category property value or the L& alias of the
```

Comment:
I think the "#" starting a line by itself here is considered bad style.

Reply:
That went away in the file re-generation (second commit).